web/packages/teleterm/src/ui/DocumentCluster/ActionButtons.tsx

```diff
@@ -233,6 +234,24 @@ function AppButton(props: {
     );
   }
 
+  if (props.app.samlApp) {
+    return (
+      <ButtonBorder
+        size="small"
+        onClick={props.connect}
```

Comment:
Ah, I copied that and forgot to change and I didn't notice it since it was using the `connectToApp` function under the hood.

Fixed reporting usage events for AWS.